### PR TITLE
More updates

### PR DESCRIPTION
**Pull request by ProPontifex**
See title

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -1025,7 +1025,7 @@ AlAoulaLaayoune.ma,Al Aoula Laayoune,قناة الأولى,,SNRT,MA,,Rabat,c/MA,
 AlAoulaMaroc.ma,Al Aoula Maroc,قناة الأولى,,SNRT,MA,,Rabat,c/MA,ara;fra;spa,general,FALSE,1962-03-03,,,http://www.alaoula.ma/,https://i.imgur.com/Q2LJMj3.png
 AlAoulaMiddleEast.ma,Al Aoula Middle East,قناة الأولى,,SNRT,MA,,Rabat,c/MA,ara;fra;spa,general,FALSE,1962-03-03,,,http://www.alaoula.ma/,https://i.imgur.com/wWKFDWf.png
 AlAqsaTV.ps,Al-Aqsa TV,قناة الأقصى,,Hamas,PS,,Gaza,c/PS,ara,general,FALSE,2006-01-09,,,,https://i.imgur.com/Z2rfrQ8.png
-Alarabiya.ae,Alarabiya,العربية,,,AE,,Dubai,c/AE,ara,news,FALSE,,,,https://www.alarabiya.net/,https://i.imgur.com/NXFkYFj.png
+Alarabiya.sa,Alarabiya,العربية,,MBC Group,SA,,Riyadh,c/AE,ara,news,FALSE,2003-03-03,,,https://www.alarabiya.net/,https://i.imgur.com/NXFkYFj.png
```
Comment (BellezaEmporium) [inline data/channels.csv:1028]: Yes... but no. The channel is indeed based in Dubai, even if MBC Group has it.

Comment (ProPontifex) [reply data/channels.csv:1028]: Thanks for that: fixed.

File: data/channels.csv [hunk 2]
```diff
@@ -3859,7 +3859,6 @@ CampoTelevision.co,Campo Television,Campo Televisión,,,CO,CO-MAG,Santa Marta,s/
 CampSpoopy.us,Camp Spoopy,,,,US,,,c/US,eng,kids,FALSE,,,,,https://i.imgur.com/8PUbPVU.png
 CampusLore.us,CampusLore,,GlewedTV,,US,US-DE,Newark,c/US,eng,sports,FALSE,,,,https://campuslore.com/,https://i.imgur.com/dCXl0rq.png
 CampusLoreSports.us,CampusLore Sports,,,,US,,,c/US,eng,sports,FALSE,,,,https://watch.plex.tv/live-tv/channel/campuslore-sports,https://i.imgur.com/MyXbINm.png
-CampusMarocTV.ma,Campus Maroc TV,قناة الجامعة المغربية,,Université Mohammed V de Rabat,MA,,Rabat,c/MA,ara;fra,education,FALSE,,,,https://campustv.ma/,https://i.imgur.com/AxhHDJb.png
```
[the remark this answers is not in the document]
Comment (ProPontifex) [reply data/channels.csv:3862]: Why? I thought I was helping. Channel highlighted reinstated.

File: data/channels.csv [hunk 3]
```diff
@@ -11996,7 +11991,6 @@ K11LCD2.us,K11LC-D2,,Comet TV,Gray Television,US,US-AZ,Prescott,s/US-AZ,eng,ente
 K11LCD3.us,K11LC-D3,,Circle,Gray Television,US,US-AZ,Prescott,s/US-AZ,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/e/e8/Circle_Network_logo.svg/512px-Circle_Network_logo.svg.png
 K11LCD4.us,K11LC-D4,,THIS,Gray Television,US,US-AZ,Prescott,s/US-AZ,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/1/12/This_TV_logo.svg/512px-This_TV_logo.svg.png
 K11LCD5.us,K11LC-D5,,Weather Now,Gray Television,US,US-AZ,Prescott,s/US-AZ,eng,weather,FALSE,,,,,https://i.imgur.com/gLU9Rfm.png
-K11UUD.as,K11UU-D,Hope Channel American Samoa,,American Samoa Adventist Media Ministry,AS,,Pago Pago,c/AS,eng;smo,religious,FALSE,2003-10-14,,,https://asamtv.org/,https://i.imgur.com/KMvhNxF.png
```
Comment (BellezaEmporium) [inline data/channels.csv:11999]: image

File: data/channels.csv [hunk 4]
```diff
@@ -18798,629 +18791,626 @@ MATV.mg,MATV,,,,MG,,Antananarivo,c/MG,fra,general,FALSE,1995-05-01,,,http://matv
...
 McKinneyTV.us,McKinney TV,,,,US,US-TX,McKinney,s/US-TX,eng,legislative,FALSE,,,,https://www.mckinneytexas.org/229/Video-Production,https://i.imgur.com/oLpPaVE.png
 McLeodsDaughters.uk,McLeod's Daughters,,Pluto TV,,UK,,,c/UK,eng,series,FALSE,,,,https://pluto.tv/en/live-tv/mcleods-daughters-gb,https://i.imgur.com/q4BBDzw.png
-MCM.fr,MCM,,,,FR,,,c/FR,fra,music,FALSE,1989-07-01,,,http://www.mcm.fr/,https://i.imgur.com/F5WNCp0.png
-MCMPop.fr,MCM Pop,MCM2,,MCM Group,FR,,,c/PT,fra,music,FALSE,2003-11-28,2014-10-01,,http://www.mcm.fr/mcm-pop/,https://i.imgur.com/j31lBL7.png
-MCMTop.fr,MCM Top,,,,FR,,,c/FR,fra,music,FALSE,2003-11-28,,,http://www.mcm.fr/mcm-top/,https://i.imgur.com/EVtWQLd.png
-MCMTop.ru,MCM Top,MCM Top,,,RU,,,c/RU,fra,music,FALSE,2015-03-01,,,https://universaltv.ru/channel/MCM-TOP,https://i.imgur.com/EVtWQLd.png
+MCM.fr,MCM,,,Groupe M6,FR,,Neuilly-sur-Seine,c/FR,fra,music,FALSE,1989-07-01,,,http://www.mcm.fr/,https://i.imgur.com/F5WNCp0.png
+MCMPop.fr,MCM Pop,MCM2,,MCM Group,FR,,Neuilly-sur-Seine,c/PT,fra,music,FALSE,2003-11-28,2014-10-01,,http://www.mcm.fr/mcm-pop/,https://i.imgur.com/j31lBL7.png
```
Comment (BellezaEmporium) [inline data/channels.csv:18898]: MCM Group -> Groupe M6

File: data/channels.csv [hunk 5]
```diff
@@ -18798,629 +18791,626 @@ MATV.mg,MATV,,,,MG,,Antananarivo,c/MG,fra,general,FALSE,1995-05-01,,,http://matv
...
+MCMTop.fr,MCM Top,,,Groupe M6,FR,,Neuilly-sur-Seine,c/FR,fra,music,FALSE,2003-11-28,,,http://www.mcm.fr/mcm-top/,https://i.imgur.com/EVtWQLd.png
```
Comment (BellezaEmporium) [inline data/channels.csv:18899]: Available in Belgium, Brazil, Monaco, Switzerland, Portugal as well.

File: data/channels.csv [hunk 6]
```diff
@@ -18798,629 +18791,626 @@ MATV.mg,MATV,,,,MG,,Antananarivo,c/MG,fra,general,FALSE,1995-05-01,,,http://matv
...
-MNetMovies3.za,M-Net Movies 3,,,MultiChoice,ZA,,,c/ZA,eng,movies,FALSE,,,,https://m-net.dstv.com/channel/m-net-movies-3-107,https://i.imgur.com/uCD89TX.png
-MNetMovies4.za,M-Net Movies 4,,,MultiChoice,ZA,,,c/ZA,eng,movies,FALSE,,,,https://m-net.dstv.com/channel/m-net-movies-4-108,https://i.imgur.com/9eUb6XS.png
-MNetMoviesMenofAction.za,M-Net Movies Men of Action,,,,ZA,,,c/ZA,eng,movies,FALSE,,,,,https://i.imgur.com/vYKO4vj.png
-MNetSport.mk,M-Net Sport,,,,MK,,,c/MK,mkd,sports,FALSE,,,,http://sport.mnet.mk/,https://i.imgur.com/7k8ar1Z.png
+MnamTV.cz,Mnam TV,Mňam TV,,S&P Broadcasting a s,CZ,,Prague,c/CZ,ces,cooking,FALSE,,,,https://imnam.cz/,https://i.imgur.com/lyfc7Ys.png
+MnauTV.cz,Mnau TV,Mňau TV,,S&P Broadcasting a s,CZ,,Prague,c/CZ,ces,,FALSE,,,,https://imnau.cz/,https://i.imgur.com/deVHJKS.png
+MNB.mn,MNB,,,Mongolian National Broadcaster,MN,,Ulaanbaatar,c/MN,mon,general,FALSE,1967-09-27,,,http://www.mnb.mn/,https://i.imgur.com/RdRd0to.png
+MNBSports.mn,MNB Sports,,,Mongolian National Broadcaster,MN,,Ulaanbaatar,c/MN,mon,sports,FALSE,,,,http://www.mnba.mn/,https://i.imgur.com/LKxcTGD.png
```
Comment (BellezaEmporium) [inline data/channels.csv:19382]: http://www.mnb.mn/

Comment (ProPontifex) [reply data/channels.csv:19382]: Corrected: thanks.

File: data/channels.csv (hunk 5 above)
Comment (ProPontifex) [reply data/channels.csv:18899]: Got it: thank you

**Review by BellezaEmporium — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**